domains/gudorlltti.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "gudorlltti",
+        "discord": "849610030431993916"
```

Comment:
there needs to be an email line, add in between "username" and "discord" this following line (assuming you dont want to expose your email):
`"email": "",`